assessmentModel/src/commonMain/kotlin/serialization/Results.kt

```diff
@@ -65,7 +65,9 @@ data class AssessmentResultObject(override val identifier: String,
                                   @SerialName("endDate")
                                   @Serializable(with = InstantSerializer::class)
                                   override var endDateTime: Instant? = null,
-                                  override val path: MutableList<PathMarker> = mutableListOf())
+                                  override val path: MutableList<PathMarker> = mutableListOf(),
+                                  override val jsonSchema: String? = null
```

Comment:
```suggestion
                                  @SerialName("$schema")
                                  override val jsonSchema: String? = null
```

Also, the default for this is:

"https://sage-bionetworks.github.io/mobile-client-json/schemas/v2/AssessmentResultObject.json"

Since this is our result object that we use for surveys.